api/lib/opentelemetry/trace/span.rb

```diff
@@ -6,42 +6,142 @@
 
 module OpenTelemetry
   module Trace
-    # Span is a mutable object storing information about the current operation execution.
+    # Span represents a single operation within a trace. Spans can be nested to
+    # form a trace tree. Often, a trace contains a root span that describes the
+    # end-to-end latency and, optionally, one or more sub-spans for its
+    # sub-operations.
+    #
+    # Once Span {Tracer#start_span is created} - Span operations can be used to
```

Comment:
Should this be `{Tracer#start_span} is created, Span` rather than `{Tracer#start_span is created} - Span`?

Reply:
This is a thing I just learned about yard, but that syntax will give you a link with the text `is created` that goes to `Tracer#start_span`.

Reply:
@mwear is correct, it's basically `{href link text}`

Reply:
TIL. 👍